.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Context
+
+<!-- Why do we need this PR? What was the reason that led you to make this change? -->
```

Comment:
> But who's going to look at the raw code? 

Exactly. 

The idea is to set a mutual expectation that we have the right information in the beginning. 

If you prefer the extra noise, that's trivial to change. Just note that you're the only one who wants it.